source/security/private-endpoints.txt

```diff
@@ -214,21 +188,161 @@ connection using either the App Services UI or the Admin API.
                  -H "Content-Type: application/json" \
                  -d '{
                    "cloud_provider_region": "<your-cloud-deployment-region>",
-                   "cloud_provider_endpoint_id": "<your-vpc-endpoint-id>"
+                   "cloud_provider_endpoint_id": "<your-vpc-endpoint-id>",
+                   "endpoint_type": "general",
+                   "comment": "<comment-about-endpoint>"
                  }'
 
-         .. step:: Modify Your VPC Private DNS
 
-            Use the AWS CLI to modify your VPC private DNS settings. This
-            ensures that your App can resolve the private endpoint's DNS name
-            to the correct IP address.
+Add a Sync Endpoint
+~~~~~~~~~~~~~~~~~~~
+
+Sync Endpoints require that you also have a General Endpoint associated with
+the Sync Endpoint.
 
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: ui
+
+      .. procedure::
+
+         .. step:: Create a VPC Interface Endpoint
+
+            a. In the App Services UI, navigate to :guilabel:`App Settings` and
+               then click the :guilabel:`Private Endpoints` tab.
+            #. Click :guilabel:`Add Private Endpoint`, then click
+               :guilabel:`Sync`.
+            #. Add your VPC ID and Subnet IDs (comma-separated list) in their
+               separate fields, then click :guilabel:`Next`.
+            #. Copy the generated :guilabel:`VPC Interface Endpoint Command`.
+               The command should look something like this:
+            
+               .. code-block:: sh
+
+                 aws ec2 create-vpc-endpoint \
+                   --vpc-endpoint-type Interface
+                   --service-name <App Services Endpoint Service Name> \
+                   --vpc-id <your-vpc-id> \
+                   --region <your-aws-deployment-region> \
+                   --subnet-ids <your-subnet-ids>
+            #. In your terminal, create a Sync VPC Endpoint with the generated
+               command.
+            #. You must also create a General VPC Endpoint using the command
+               App Services generates for the General Endpoint.
+         
+         .. step:: Add the Private Endpoints to Your App
+
+            Now that you've created the interface endpoint in your VPC, you
+            need to add it to your App. 
```

Comment:
Are we creating more than one VPC endpoint in the step above - i.e. one for Sync and one for General? If so, we should probably match the plural here - i.e.

```suggestion
            Now that you've created the interface endpoints in your VPC, you
            need to add them to your App. 
```